services/gateway/sources/version1/newsfeed.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const newsfeed = require('./mappings/newsfeed');
+
+module.exports = {
+	type: 'moleculer',
+	method: 'newsfeed.articles',
+	params: {
+		source: '=,string',
+		offset: '=,number',
+		limit: '=,number',
+	},
+	definition: {
+		data: ['data', newsfeed],
+		meta: {
+			count: '=,number',
```

Comment:
Are we not including the `limit`, `offset` and `source` information in `response.meta`?

Reply:
Actually, I followed the specification document as well as the description of the issue. Both specify `meta` to include only `count`.

Reply:
Not returning the `offset` value would break the pagination support on the endpoint.

@MichalTuleja could you please confirm if we should include the `limit`, `offset` and `source` information (or a subset thereof) in `response.meta`?

Reply:
Version 1 API does not support pagination at all. The default limit is 10, that was requested by the UI at the time of design/development.